.github/workflows/gha_linkcheck.yml

```diff
@@ -0,0 +1,28 @@
+name: Linkcheck
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  packages:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.x"
+
+    - name: Install
+      run: |
+        python -m pip install pytest-check-links
+
+    - name: Check Links
+      run:  python -m pytest --check-links
```

Comment:
This is the line. I never use the pytest --check-links before and I could not find a document that mention the usage without providing a filename. Would the command without filename just check links for all files in the directories? Thanks!

Reply:
If someone with the rights to this repo approves the GHA run you'll see something like this: 

https://github.com/ocefpaf/CEFI-info-hub-list/actions/runs/5189447297/jobs/9354473036

The tests look for html, rst, md, and ipynb extensions from the start point. We need to hardcode json for this repo.

Reply:
Thanks for the link to the test! Do you think it would work for adding this job (link testing) as a subsequent job to the json => md GitHub action? based on the test you shown it is testing the link on the created md file probably from the previous version without the inclusion of the link in the json file in the new PR submit.

Reply:
To make it clearer of what I meant. This means the workflow would be 
1. Github Action job1 : json convert the md file
2. Github Action job2 : link check for the md file
This way when there is a new link added to the json file in a PR the link check will be able to check the link link as well.

Reply:
This test, ideally, should scan all files. With that said, b/c there is a creation of a md from json (I did not know the details until now), we can "fold" this GHA into the json->md one to avoid an overkill of checks. I'll look into that and implement it in a moment.

Reply:
Ah! thank you for the confirmation. I can actually do the changes from your branch if you are busy. But feel free to make the changes if you prefer to do it! I probably cannot get to this change today.